app/src/main/kotlin/cz/covid19cz/erouska/ui/exposure/ExposuresFragment.kt

```diff
@@ -30,6 +30,9 @@ class ExposuresFragment : BaseFragment<FragmentExposuresBinding, ExposuresVM>(
         symptoms_text.text = AppConfig.mainSymptoms
         spread_text.text = AppConfig.spreadPrevention
         earlier_exposures_text.text = AppConfig.earlierExposures
+        earlier_exposures_button.text = AppConfig.earlierExposures
```

Comment:
Podtržítka do Kotlin/Java kódu nepatří, id views by měly být camelCase.

Reply:
Absolutně souhlasím, ale když jsem přišel na projekt, tak jsem viděl všude v XML snake_case a nechtěl jsem bourat zajetou konvenci

Reply:
ok, tak to jsou nějací kostlivci z minulosti, necháme to tedy na nějaký pozdější refactor... #nikdy